spec/unit/finder_methods_spec.rb

```diff
@@ -47,4 +55,34 @@ def initialize
                              "_query_:\"{!raw f=library_id}four\"" }
     end
   end
+
+  describe "#find_in_batches" do
+    let(:docs) {
```

Comment:
Minor quibble: we use do/end for multiline blocks and {} for single-line blocks.